examples/vic-gov-au/tide/tide.config.js

```diff
@@ -11,6 +11,15 @@ const tideConfig = {
 
   mapping: {
     // landingPageComponents: {}
+  },
+
+  modules: {
+    'event': {
+      moduleHook: true,
+      config: true,
+      mappingFilters: true,
+      pageTypes: true
+    }
```

Comment:
This is the proposal to let custom sites register their modules. I copied our core `event` modules to custom site for testing at this stage.